### ISSUE TITLE
[PABP] Allow User to pull Email from Content Library

### ISSUE THREAD

**Issue by daver78** [coa0329/BhaangWeb#175]
![image](https://user-images.githubusercontent.com/7561473/32013829-19c84e7a-b971-11e7-8fa7-00df1d47ecaf.png)

Need to find an easy, elegant way to allow them to start from a template or an existing Email in the Content Library.

**Comment by paagla123** [comment]
@paweenwich Let's discuss this one more time.

**Comment by paagla123** [comment]
@paweenwich, I have discussed with @daver78 . While he agreed that we can lower the priority of saving htmls in the asset library from da Vinci, there is another requirement of people wanting to upload an externally generated landing page (which will be static - no edits) into da Vinci and use da Viinci to simplify the scheduling function. So we should keep this requirement alive.

**Comment by paagla123** [comment]
@daver78 will you define a place in the UI where this can be done? Both PABP and PAEB - landing pages and emails.

**Comment by daver78** [comment]
@paagla123 What is "facify"?  

In #187, I've added the ability to insert your own HTML.  How is this Issue different?  Should we close one of them?

**Comment by daver78** [comment]
@paagla123 Looks like facify is a word but not sure how to apply in this context:

![image](https://user-images.githubusercontent.com/7561473/32351289-15bd8f92-bfda-11e7-8bd1-b9d965895193.png)

**Comment by paagla123** [comment]
@daver78 => Fancify.. Its a word I made up. Facify was a typo. Are yiu ok with the dropdown suggested by @paweenwich ?

**Comment by daver78** [comment]
@paagla123 Where does the drop-down go?

**Comment by paagla123** [comment]
@daver78 its coming, flight is delayed!
Its assigned to @jiewsone but I don;t think we will address it until he is done with his current assignment.
@jiewsone please put a fake dropdown in the  UI stating Read from Asset Library (Coming soon)

**Comment by daver78** [comment]
@paagla123 and @jiewsone For the sake of the User, I would prefer that for now, we do **not** add another pull-down.  Instead, let's put all the options inside the existing pull-down, group-by "Templates" or "Asset Library" like this:

_Templates
- Template 1
- Template 2
Asset Library
- HTML 1
- HTML 2
- HTML n_

I believe this reduces clutter and cognitive overhead for the User.  There's only one place to look.

Is ok?

**Comment by paagla123** [comment]
@daver78 Yep good. @jiewsone I think this will make things easier for you.
